tests/test_wave_01.py

```diff
@@ -122,8 +121,9 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][0]["rating"] is RATING_1
+    assert updated_data["watched"][0]["genre"] is GENRE_1
```

Comment:
In this case where the code is checking individual keys of the dictionary, I recommend including all of the relevant keys.  Also, this was an error on our side, but "is" can cause some issues and "==" is the preferred way to check equality.
```suggestion
    assert updated_data["watched"][0]["rating"] == RATING_1
    assert updated_data["watched"][0]["genre"] == GENRE_1
    assert updated_data["watched"][0]["title] == MOVIE_TITLE_1
```